test/common/ContractInteractor.test.ts

```diff
@@ -1,27 +1,28 @@
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
 
-import { RelayHubInstance, TestVersionsInstance } from '../../types/truffle-contracts'
-import { RelayClient } from '../../src/relayclient/RelayClient'
+import { RelayHubInstance } from '../../types/truffle-contracts'
 import { HttpProvider } from 'web3-core'
 import { ProfilingProvider } from '../../src/common/dev/ProfilingProvider'
-import ContractInteractor from '../../src/relayclient/ContractInteractor'
-import { configureGSN } from '../../src/relayclient/GSNConfigurator'
+import ContractInteractor from '../../src/common/ContractInteractor'
 import { PrefixedHexString } from 'ethereumjs-tx'
 import Transaction from 'ethereumjs-tx/dist/transaction'
 import { constants } from '../../src/common/Constants'
 import { createClientLogger } from '../../src/relayclient/ClientWinstonLogger'
 import RelayRequest from '../../src/common/EIP712/RelayRequest'
 import { deployHub } from '../TestUtils'
+import VersionsManager from '../../src/common/VersionsManager'
+import { gsnRuntimeVersion } from '../../src/common/Version'
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const { expect } = chai.use(chaiAsPromised)
 
-const TestVersions = artifacts.require('TestVersions')
 const TestPaymasterConfigurableMisbehavior = artifacts.require('TestPaymasterConfigurableMisbehavior')
 const StakeManager = artifacts.require('StakeManager')
 const Penalizer = artifacts.require('Penalizer')
 
 contract('ContractInteractor', function (accounts) {
+  /*
```

Comment:
whyh comment out skipped tst?

Reply:
Reintroduced this test now. See "should throw if wrong contract paymaster version".

Reply:
If I understand correctly the version we care about is the Paymaster version now, not the Hub version. At least there was no place where we would've requested the version form the hub.

Reply:
yes. there should be (offline?) test that the paymaster matches the hub. its more of a job for the paymaster itself (or paymaster's owner), not the client's